nucypher/cli/options.py

```diff
@@ -42,6 +42,7 @@
 option_event_name = click.option('--event-name', help="Specify an event by name", type=click.STRING)
 option_federated_only = click.option('--federated-only/--decentralized', '-F', help="Connect only to federated nodes", is_flag=True, default=None)
 option_force = click.option('--force', help="Don't ask for confirmation", is_flag=True)
+option_gas_price = click.option('--gas-price', help="Use this gas price (in GWEI)", type=DecimalRange(min=0))
```

Comment:
Perhaps start of with a new click type here for gwei,

Reply:
✅